README.md

```diff
@@ -182,6 +186,11 @@ VARIABLE_WITH_DEFAULT: !var:default='defaultvalue' path/to/variable
 
     This flag can be useful when the underlying system that's going to be using the values implements defaults. For example, when using summon as a bridge to [confd](https://github.com/kelseyhightower/confd).
 
+* `--all-provider-versions, -V` List of all of the providers in the default path and their versions
```

Comment:
Codeclimate is complaining about line length

Reply:
updated.